proto/engine/v1/execution_engine.proto

```diff
@@ -132,10 +132,13 @@ message Withdrawal {
 	// Withdrawal index for accounting purposes
 	uint64 withdrawal_index = 1;
 
+	// Validator index for the withdrawal
+	uint64 validator_index = 2 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v3/consensus-types/primitives.ValidatorIndex"];
+
 	// The execution address receiving the funds
-	bytes execution_address = 2 [(ethereum.eth.ext.ssz_size) = "20"];
+	bytes execution_address = 3 [(ethereum.eth.ext.ssz_size) = "20"];
 
 	// The withdrawn amount in Gwei
-	uint64 amount = 3;
+	uint64 amount = 4;
 }
```

Comment:
Do not change IDs. This is not going to be backwards compatible with the previous version.

Reply:
what do you mean? there's no previous version, the spec is in flux

Reply:
What I mean is that we should not be changing proto IDs. In this case, it seems like we can call it out as something that has not be implemented yet so it is safe to change, but it has to be explicit so that we don't develop the habit of changing proto IDs like this.